streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -46,26 +55,27 @@ public void configure() throws IOException, Exception {
 	@VisibleForTesting
 	FhirConverter createFhirConverter(CamelContext camelContext) throws IOException, Exception {
 		FhirContext fhirContext = FhirContext.forDstu3();
-		String fhirBaseUrl = System.getProperty("openmrs.serverUrl") + System.getProperty("openmrs.fhirBaseEndpoint");
-		OpenmrsUtil openmrsUtil = new OpenmrsUtil(fhirBaseUrl, System.getProperty("openmrs.username"),
-		        System.getProperty("openmrs.password"), fhirContext);
-		FhirStoreUtil fhirStoreUtil = FhirStoreUtil.createFhirStoreUtil(System.getProperty("fhir.sinkPath"),
-		    System.getProperty("fhir.sinkUsername"), System.getProperty("fhir.sinkPassword"),
-		    fhirContext.getRestfulClientFactory());
+		DebeziumArgs params = DebeziumArgs.getInstance();
```

Comment:
Is there a reason to recreated `params` for each call of this function? I mean, why not make `params` an instance variable and initialize it in the constructor only once (i.e., move this and next line to the constructor)?

Also is there any reason to define `args` as a static variable while you are initializing it in the constructor? Of course, if you do my first suggestion you don't need to keep `args` anymore, but I was just curious.